src/server/pegasus_server_write.cpp

```diff
@@ -170,5 +136,77 @@ void pegasus_server_write::request_key_check(int64_t decree,
     }
 }
 
+pegasus_server_write::single_put_rpc_map pegasus_server_write::_single_put_methods = {
+    {dsn::apps::RPC_RRDB_RRDB_MULTI_PUT,
+     [](pegasus_server_write *server_write, dsn::message_ex *request) -> int {
+         return server_write->multi_put(request);
+     }},
```

Comment:
```suggestion
    {dsn::apps::RPC_RRDB_RRDB_MULTI_PUT,
     [this](dsn::message_ex *request) -> int {
         return multi_put(request);
     }},
```

Can we use "this"?

Reply:
No, because _single_put_methods is static. I don't want to make it to be a non-static private member.